pool/doc.go

```diff
@@ -5,6 +5,12 @@ The main component is Pool type. It is a virtual connection to the network
 and provides methods for executing operations on the server. It also supports
 a weighted random selection of the underlying client to make requests.
 
+Pool has an auto-session mechanism for object operations. It is enabled by default.
+The mechanism allows to manipulate objects like upload, download, delete, etc, without explicit session passing.
+This behavior may be disabled per request by calling IgnoreSession() on the appropriate Prm* argument.
+Note that if auto-session is disabled, the user MUST provide the appropriate session manually for PUT and DELETE object operations.
```

Comment:
is it possible in the current pool to send a signed object? it does not require a session theoretically

Reply:
I consider no one tried it

Reply:
according to that [thread](https://github.com/nspcc-dev/neofs-sdk-go/pull/434#discussion_r1214676012) that PR (and the #449) should make it possible but it seems like it done